docs/imaging/dlc.rst

```diff
@@ -0,0 +1,114 @@
+.. _Imaging dlc:
+
+=========================
+Imaging: DLC
+=========================
+
+`DeepLabCut <http://www.mackenziemathislab.org/deeplabcut>`_ is a software package for inferring motion capture information from tracking video. Whereas Optitrack makes use of one infra-red reflectors attached to the subject(s), DeepLabCut instead uses a deep neural network trained on human-labelled data of the subject(s), allowing for less interference with subject behaviour. 
+
+
+
+Schema
+------------
+
+The DLC implementation in the imaging pipeline is shown below
+
+.. figure:: /_static/imaging/dlc_schema.png
+   :alt: DLC in the Imaging schema
+   
+   DLC in the Imaging schema
+
+* ``DLCTrackingType``
+
+  Keeps track of the kinds of tracking to which DLC can be assigned. At present, DLC data is only supported for *openfield* tasks
+
+* ``DLCModel``
+
+  Contains the full specification of a trained DLC model used in prediction. Due to the way DLC is designed, this data is derived from several separate files.
+  When creating a new Method, please pay careful attention to how body parts are named (see ``TrackedBodyPart``)
+
+* ``DLCTrackingProcessingMethod``
```

Comment:
I think this is a good description (Thinh should confirm). But maybe make distinction between `DLCTrackingProcessingMethod` and `DLCProcessingMethod` more clear, e.g. DLCProcessingMethod combines Model on the one hand with applicable (post)processing methods on the other ... or similar.